esrally/rally.py

```diff
@@ -169,6 +174,51 @@ def positive_number(v):
         default=""
     )
 
+    provision_parser.add_argument(
+        "--team-repository",
+        help="Define the repository from where Rally will load teams and cars (default: default).",
+        default="default")
+    provision_parser.add_argument(
+        "--car",
+        help="Define the car to use. List possible cars with `%s list cars` (default: defaults)." % PROGRAM_NAME,
+        default="defaults")  # optimized for local usage
+    provision_parser.add_argument(
+        "--car-params",
+        help="Define a comma-separated list of key:value pairs that are injected verbatim as variables for the car.",
+        default=""
+    )
+    provision_parser.add_argument(
+        "--distribution-version",
+        help="Define the version of the Elasticsearch distribution to download. "
+             "Check https://www.elastic.co/downloads/elasticsearch for released versions.",
+        default="")
+    provision_parser.add_argument("--node-ids",
```

Comment:
I think this subcommand should only ever support provisioning one node at a time. Provisioning multiple nodes should be handled in the layer above.

Reply:
Sounds good to me.